.github/workflows/build-docker-images.yml

```diff
@@ -10,29 +10,55 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Build endpoint
-        uses: docker/build-push-action@v1
+
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
```

Comment:
What does this do?

Reply:
It is installing QEMU binaries to support building for ARM.

The suggestion to use this came from https://github.com/docker/buildx/issues/495#issuecomment-918925854, prior to including this I was getting many [errors in my local fork](https://github.com/WesleyRosenblum/quic-network-simulator/runs/5623023112?check_suite_focus=true#step:5:433), such as:
```
#15 28.06 Error while loading /usr/sbin/dpkg-split: No such file or directory
#15 28.07 Error while loading /usr/sbin/dpkg-deb: No such file or directory
#15 28.07 dpkg: error processing archive /tmp/apt-dpkg-install-Gsh52m/00-libssl1.1_1.1.1f-1ubuntu2.12_arm64.deb (--unpack):
#15 28.07  dpkg-deb --control subprocess returned error exit status 1
```